### PR TITLE
Fix input port of CBN for localized build

### PR DESCRIPTION
**Pull request by sharadkjaiswal**
### Purpose
Fixes http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7588
- Unbound variable name was obtained from warning message and for localized builds the warning messages are localized, so it 's hard to parse the warning message correctly to get the unbound variable name.
- Instead use UnboundVariableSymbolNode property of WarningEntry object to get the unbound variable name.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] The level of testing this PR includes is appropriate

### Reviewers

- [x] @aparajit-pratap  

### FYIs
@junmendoza 
@monikaprabhu

### CODE REVIEW

**Comment by aparajit-pratap** [comment]
@sharadkjaiswal did you run automation on this branch to check for any regression?

**Comment by sharadkjaiswal** [comment]
Yes, using self service CI, all language and Dynamo tests are passing.

**Comment by aparajit-pratap** [comment]
LGTM
